src/NextGen/dataRepository.ts

```diff
@@ -0,0 +1,14 @@
+// QUESTION: Should we adopt BaseMessage like Native?
```

Comment:
Can you remind me the differences?

Reply:
A BaseMessage seems to have an mpid, session id, timestamps, attributes and some event level meta data (like Message Type) and it's somewhat similar to our BaseEvent.

For more specifics: https://github.com/mParticle/mparticle-android-sdk/blob/main/android-core/src/main/java/com/mparticle/internal/messages/BaseMPMessage.java

Reply:
From recent conversation: Web should start using Messages for "Storage" in the Data Repository. Create a new ticket.